src/deadline/cinema4d_submitter/cinema4d_render_submitter.py

```diff
@@ -426,7 +426,7 @@ def on_create_job_bundle_callback(
         job_setup_widget_type=SceneSettingsWidget,
         initial_job_settings=render_settings,
         initial_shared_parameter_values={
-            "CondaPackages": "",
+            "CondaPackages": f"cinema4d={str(c4d.GetC4DVersion())[:4]}.* cinema4d-openjd=0.*",
```

Comment:
Is there a reason for the adaptor being 0.* vs 0.MINOR.* like most of the other integrations do? e.g. [houdini](https://github.com/aws-deadline/deadline-cloud-for-houdini/blob/244bc7130999f38e5d408b0e845939411b09d2a4/src/deadline/houdini_submitter/python/deadline_cloud_for_houdini/queue_parameters.py#L150)

Reply:
Updated to match the format of other packages and follow semantic versioning!